src/stories/RedPackets.tsx

```diff
@@ -110,7 +111,7 @@ storiesOf('Plugin: Red Packets', module)
                         mnemonic: ['mem'],
                         name: 'Wallet Name',
                         passphrase: 'password',
-                        eth_balance: BigInt(2 * 10 ** 18),
+                        eth_balance: new BigNumber(2).multipliedBy(new BigNumber(10).pow(18)), // 2 * (10 ** 18)
```

Comment:
Why not `2 * 10 ** 18`?

Reply:
see https://www.npmjs.com/package/bignumber.js#use 
```
new BigNumber(99999999999999999999)       // '100000000000000000000'
```

Reply:
(The original code is actually unsafe)